<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Workspace</class>
 <widget class="QWidget" name="Workspace">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Workspace</string>
  </property>
  <layout class="QHBoxLayout" name="playerLayout" stretch="2,1">
   <item>
    <widget class="QWidget" name="videoWidget" native="true">
     <property name="focusPolicy">
      <enum>Qt::FocusPolicy::ClickFocus</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="sideBar" native="true">
     <layout class="QVBoxLayout" name="sideLayout" stretch="0,1">
      <item>
       <widget class="QWidget" name="testButtonBar" native="true">
        <layout class="QHBoxLayout" name="testBarLayout">
         <item>
          <widget class="QPushButton" name="loadTestVideoButton">
           <property name="text">
            <string>Load Video</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="padding">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWebEngineView" name="dialogueWebView">
        <property name="url">
         <url>
          <string>about:blank</string>
         </url>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
